ports/servoshell/keyutils.rs

```diff
@@ -20,100 +20,339 @@ pub const CMD_OR_ALT: Modifiers = Modifiers::META;
+        NamedKey::FavoriteClear3 => Key::FavoriteClear3,
+        NamedKey::FavoriteRecall0 => Key::FavoriteRecall0,
+        NamedKey::FavoriteRecall1 => Key::FavoriteRecall1,
+        NamedKey::FavoriteRecall2 => Key::FavoriteRecall2,
+        NamedKey::FavoriteRecall3 => Key::FavoriteRecall3,
+        NamedKey::FavoriteStore0 => Key::FavoriteStore0,
+        NamedKey::FavoriteStore1 => Key::FavoriteStore1,
+        NamedKey::FavoriteStore2 => Key::FavoriteStore2,
+        NamedKey::FavoriteStore3 => Key::FavoriteStore3,
+        NamedKey::Guide => Key::Guide,
+        NamedKey::GuideNextDay => Key::GuideNextDay,
+        NamedKey::GuidePreviousDay => Key::GuidePreviousDay,
+        NamedKey::Info => Key::Info,
+        NamedKey::InstantReplay => Key::InstantReplay,
+        NamedKey::Link => Key::Link,
+        NamedKey::ListProgram => Key::ListProgram,
+        NamedKey::LiveContent => Key::LiveContent,
+        NamedKey::Lock => Key::Lock,
+        NamedKey::MediaApps => Key::MediaApps,
+        NamedKey::MediaAudioTrack => Key::MediaAudioTrack,
+        NamedKey::MediaLast => Key::MediaLast,
+        NamedKey::MediaSkipBackward => Key::MediaSkipBackward,
+        NamedKey::MediaSkipForward => Key::MediaSkipForward,
+        NamedKey::MediaStepBackward => Key::MediaStepBackward,
+        NamedKey::MediaStepForward => Key::MediaStepForward,
+        NamedKey::MediaTopMenu => Key::MediaTopMenu,
+        NamedKey::NavigateIn => Key::NavigateIn,
+        NamedKey::NavigateNext => Key::NavigateNext,
+        NamedKey::NavigateOut => Key::NavigateOut,
+        NamedKey::NavigatePrevious => Key::NavigatePrevious,
+        NamedKey::NextFavoriteChannel => Key::NextFavoriteChannel,
+        NamedKey::NextUserProfile => Key::NextUserProfile,
+        NamedKey::OnDemand => Key::OnDemand,
+        NamedKey::Pairing => Key::Pairing,
+        NamedKey::PinPDown => Key::PinPDown,
+        NamedKey::PinPMove => Key::PinPMove,
+        NamedKey::PinPToggle => Key::PinPToggle,
+        NamedKey::PinPUp => Key::PinPUp,
+        NamedKey::PlaySpeedDown => Key::PlaySpeedDown,
+        NamedKey::PlaySpeedReset => Key::PlaySpeedReset,
+        NamedKey::PlaySpeedUp => Key::PlaySpeedUp,
+        NamedKey::RandomToggle => Key::RandomToggle,
+        NamedKey::RcLowBattery => Key::RcLowBattery,
+        NamedKey::RecordSpeedNext => Key::RecordSpeedNext,
+        NamedKey::RfBypass => Key::RfBypass,
+        NamedKey::ScanChannelsToggle => Key::ScanChannelsToggle,
+        NamedKey::ScreenModeNext => Key::ScreenModeNext,
+        NamedKey::Settings => Key::Settings,
+        NamedKey::SplitScreenToggle => Key::SplitScreenToggle,
+        NamedKey::STBInput => Key::STBInput,
+        NamedKey::STBPower => Key::STBPower,
+        NamedKey::Subtitle => Key::Subtitle,
+        NamedKey::Teletext => Key::Teletext,
+        NamedKey::VideoModeNext => Key::VideoModeNext,
+        NamedKey::Wink => Key::Wink,
+        NamedKey::ZoomToggle => Key::ZoomToggle,
+
```

Comment:
nit: maybe you can remove extra space here

Reply:
Leaving this space, seperates the unusual cases (NamedKey::Space, default) from the large block of uninteresting A=>A cases above.